test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Execution/TestMethodRunnerTests.cs

```diff
@@ -111,6 +116,106 @@ public void Cleanup()
         }
 
         [TestMethodV1]
+        public void IgnoreAttributeOnTestClassIgnoresTestAndPrintsClassIgnoreMessageIfProvided()
```

Comment:
shouldn't all these method names start with `ExecuteShould` instead since that is what you are acting on below?